tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -209,3 +218,34 @@ func createDownstreamCluster(client *rancher.Client, clusterType string) (*manag
 
 	return clusterObject, steveObject, testClusterConfig, nil
 }
+
+func createGlobalRole(client *rancher.Client, inheritedClusterrole []string) (*v3.GlobalRole, error) {
+	globalRole.Name = namegen.AppendRandomString("testgr")
+	globalRole.InheritedClusterRoles = inheritedClusterrole
+	createdGlobalRole, err := rbac.CreateGlobalRole(client, &globalRole)
+	return createdGlobalRole, err
+}
+
+func createGlobalRoleAndUser(t *testing.T, client *rancher.Client, inheritedClusterrole []string) (*management.User, error) {
+	globalRole, err := createGlobalRole(client, inheritedClusterrole)
+	require.NoError(t, err)
+
+	createdUser, err := users.CreateUserWithRole(client, users.UserConfig(), standardUser, globalRole.Name)
+	require.NoError(t, err)
+
+	return createdUser, err
+
+}
+
+func getErrorMessage(err, startText, endText string) (actualError string) {
```

Comment:
Can we delete this function since this is no longer used?

Reply:
deleted. thanks!